geocoding.go

```diff
@@ -27,6 +27,12 @@ type ReverseGeocodeRequest struct {
 	Types    Types  `json:"types,omitempty"`
 }
 
+type ForwardGeocodeBatchRequest []ForwardGeocodeRequest
```

Comment:
Consider renaming `ForwardGeocodeBatchRequest` to something to indicate of its bulk processing nature for clarity. How about `BulkForwardGeocodeRequest`?